pkg/hostagent/port_darwin.go

```diff
@@ -8,13 +8,27 @@ import (
 	"path/filepath"
 	"strconv"
 	"strings"
+	"syscall"
 
 	"github.com/lima-vm/lima/pkg/bicopy"
 	"github.com/lima-vm/lima/pkg/guestagent/api"
 	"github.com/lima-vm/sshocker/pkg/ssh"
 	"github.com/sirupsen/logrus"
 )
 
+// Default nofile limit in the standard terminal in macOS is 256.
+// It means that there are only ~240 connections available from the host to the vm.
+// That function increases the nofile limit for child processes, especially the ssh process
+//
+// More about limits in go: https://go.dev/issue/46279
+func init() {
+	var limit syscall.Rlimit
+	if err := syscall.Getrlimit(syscall.RLIMIT_NOFILE, &limit); err == nil && limit.Cur != limit.Max {
+		limit.Cur = limit.Max
+		_ = syscall.Setrlimit(syscall.RLIMIT_NOFILE, &limit)
```

Comment:
Maybe the error should be logged via `logrus.Debug`?

Reply:
I also wonder this should be moved to https://github.com/lima-vm/lima/blob/master/cmd/limactl/hostagent.go , as it affects reverse-sshfs too IIUC.

Reply:
(It covers non-Darwin too)

Reply:
Yes, it affects reverse-sshfs too.

I moved the code to `hostagent.go` and make the fuction comment more wide.
Also I added debug logging if an error occurs